doc/api/n-api.md

```diff
@@ -3155,6 +3155,21 @@ support it:
 * If the function is not available, provide an alternate implementation
   that does not use the function.
 
+### napi_adjust_external_memory
+<!-- YAML
+added: REPLACEME
+-->
+```C
+NAPI_EXTERN int64_t napi_adjust_external_memory(napi_env env,
+                                         int64_t change_in_bytes);
```

Comment:
I think the parameters should be aligned here, that probably means the first one will have to be on a new line. I'm surprised the linter did not complain about this.